src/entities/addLiquidityNested/index.ts

```diff
@@ -33,6 +33,9 @@ export class AddLiquidityNested {
                 return addLiquidity.query(input, nestedPoolState);
             }
             case 3: {
+                if (input.chainId === 43114) {
+                    throw new Error('Avax not supported for nested operations');
```

Comment:
Could be confusing since `AVAX` is also a ticker :sweat_smile: 

Maybe use the new SDK error like..

```
                if (input.chainId === ChainId.AVALANCHE) {
                    throw new SDKError(
                        'Input Validation',
                        'Add Liquidity Nested',
                        'Balancer V3 does not support this operation on Avalanche',
                    );
                }
```